### ISSUE TITLE
add web demo/models/datasets to SIGGRAPH 2022 organization on Hugging Face

### ISSUE THREAD

**Issue by AK391** [kbrodt/sketch2pose#2]
Hi, congrats for the acceptance at SIGGRAPH 2022. We are having an event on Hugging Face for SIGGRAPH 2022, where you can submit spaces(web demos), models, and datasets for papers for a chance to win prizes. The Hub offers free hosting and would make your work more accessible to the rest of the community. Hugging Hub works similar to github where you can push models/dataset/demos to user profiles or organization accounts. For siggraph you can add the models/datasets/demos to this organization:

https://huggingface.co/SIGGRAPH2022

after joining the organization using this link: https://huggingface.co/organizations/SIGGRAPH2022/share/lKvMytVEyvqyWBIbfrcZTeBhNQLXklhIHG

let me know if you need any help with the above steps, thanks

**Comment by kbrodt** [comment]
Hi, thanks! I have added the [demo](https://huggingface.co/spaces/SIGGRAPH2022/sketch2pose), though it is not full due to lack of GPU.

**Comment by AK391** [comment]
@kbrodt thanks for adding the demo although it is with the streamlit sdk, have you tried gradio: https://gradio.app/, it has some nice features like queueing, concurrency, and is used in apps like min-dalle: https://huggingface.co/spaces/kuprel/min-dalle

**Comment by kbrodt** [comment]
@AK391 I'm trying to use gradio. Is it possible to get access to GPU resources for full demo and to cache results for all examples?

**Comment by AK391** [comment]
@kbrodt yes, you can request gpu access like this https://huggingface.co/spaces/THUDM/CogVideo/discussions/2, usually it is a t4 gpu, for gradio examples are cached by default on spaces with the setting cache_examples=True, I see you are using Interface API so the docs are here: https://gradio.app/docs/#interface-header

**Comment by kbrodt** [comment]
@AK391 I published [demo](https://huggingface.co/spaces/SIGGRAPH2022/sketch2pose) based on gradio sdk. Also I [requested](https://huggingface.co/spaces/SIGGRAPH2022/sketch2pose/discussions/1) a GPU

**Comment by AK391** [comment]
@kbrodt awesome, thanks, I will also followup with the team on the gpu request

**Comment by kbrodt** [comment]
Thanks! I had tested and anythinks woks fine.

**Comment by Cicistawberry** [comment]
May I ask why I can only run sample bitmap images on demo? I can't upload a new bitmap image to run myself. T T

**Comment by Cicistrawberry** [comment]
@kbrodt May I ask why I can only run sample bitmap images on demo? I can't upload a new bitmap image to run myself.

**Comment by kbrodt** [comment]
I've tested and it works, though on CPU it takes about 5 min 
![image](https://user-images.githubusercontent.com/20697252/204024789-9af7f13d-6e1a-43f5-bf9e-203251bb7ca0.png)

**Comment by Cicistrawberry** [comment]
> I've tested and it works, though on CPU it takes about 5 min ![image](https://user-images.githubusercontent.com/20697252/204024789-9af7f13d-6e1a-43f5-bf9e-203251bb7ca0.png)

I tried again and made it, too. Thank you very much!!!

**Comment by Xushuolin** [comment]
> @kbrodt May I ask why I can only run sample bitmap images on demo? I can't upload a new bitmap image to run myself.

me too
